docs/source/install/overview.rst

```diff
@@ -0,0 +1,62 @@
+Overview: Single-box Reference Deployment
+=========================================
+
+.. todo:: short intro paragraph: this is reference deployment for single box, follow installation
+    guide to get here.
+
+.. figure:: https://docs.google.com/drawings/d/1X6u8BB9bnWkW8C81ERBvjIKRfo9mDos4XEKeDv6YiF0/pub?w=960&amp;h=720
```

Comment:
No @manasdk, I do understand why we end up where we end up. There was a looong chain of decisions and each of them we're more or less valid. Dunno, may be "overly" is no the right word... "sufficiently complicated". Anyway, the point is, st2 is huge and we should not hide this fact from our users.